README.md

```diff
@@ -176,14 +176,14 @@ And then run a workload:
 ```sh
 ./scripts/genny run                                                 \
     --workload-file       ./src/workloads/scale/InsertRemove.yml    \
-    --metrics-format      csv                                       \
-    --metrics-output-file build/genny-metrics.csv                   \
     --mongo-uri           'mongodb://localhost:27017'
 ```
 
-Logging currently goes to stdout, and time-series metrics data is
-written to the file indicated by the `-o` flag (`./genny-metrics.csv`
-in the above example).
+Logging currently goes to stdout, and in this example, metrics data is
+written to `./build/genny-metrics.csv`. The metrics format and output
```

Comment:
Similar to my comment below, I think of the metrics format as something for advanced users, so personally I would leave its description in the HelloWorld workload file instead of the README. Whatdoyathink?

Reply:
Yeah that sounds reasonable; I'll just modify the README to say where the results will be outputted.